src/extension/rn-extension.ts

```diff
@@ -1,6 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
+import * as fs from "fs";
+
+try {
+    fs.statSync(`${__filename}.map`); // We check if source maps are available
+    /* tslint:disable:no-var-requires */
+    require("source-map-support").install(); // If they are, we enable stack traces translation to typescript
+    /* tslint:enable:no-var-requires */
+} catch (exceptions) {
+    // If something goes wrong, we just ignore the errors
```

Comment:
My understanding is that this is intended to be debug-mode-only, and in a production scenario it is expected to throw an exception because `statSync` will fail to find the file. If we want to log exceptions, we should check for the map files in a way that won't throw an exception when they are missing.